api/src/Page/Sample.php

```diff
@@ -1984,24 +1985,45 @@ function _update_sample_group() {
             }
         }
 
-        function _add_sample_to_group() {
-            if (!$this->has_arg('BLSAMPLEID')) $this->_error('No sample specified');
+        function _add_sample_to_group () {
+            $sample = array('BLSAMPLEID' => $this->arg('BLSAMPLEID'), 'BLSAMPLEGROUPID' => $this->arg('BLSAMPLEGROUPID'));
```

Comment:
I think we should have guards at the top. 
So check for presence of BLSAMPLEGROUPID and BLSAMPLEID here with has_arg(), and return error if they are not found.

Reply:
fixed